website/docs/fun/counting_v2.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Counting System
+---
+
+This command controls the counting game and the statistics command.
+
+## Trigger
+
+**Type:** `Regex`<br />
+**Trigger:** `.+`
```

Comment:
Note that `.+` does not match text split over multiple lines. I have not examined your code thoroughly but it seems like this could be a concern; for instance, would
```
1
<random text here>
```
be marked as invalid, assuming `1` is the next number?

Note also that this does not trigger on image-only posts, which might be abused? Not sure.

Reply:
Your example is valid and correct given that the number or expression on the first line evaluates to the next number.
I'm unsure how image-only posts could be used to abuse the system. My intent in many of my commands has been to allow "outside conversation" within the channel while not disrupting the game at hand. To do this, but also allow for such a wide range of input, I chose this trigger.

Reply:
Fair enough. That is a significant difference from most counting systems I have seen in the wild, so I think it is worth mentioning explicitly in the description of this command (that unrelated content is permitted.)

Reply:
As messages that do include text content will trigger the command and an output if all characters do not match the available numerals, operators, or functions, maybe my claim for "outside conversation" is invalid for this command.
For example, if a user were to post "1+1 equals 2", this would be taken as invalid and a message would be output to show that.
So while messages without text will be ignored/permitted, messages with text while maybe unrelated, will not be entirely ignored. They will not affect the system or modify internal values.
So where I am now is how to word this in the description, or if it's even worth including.

Reply:
We should certainly document that the command does not reject invalid inputs by deleting messages, since many other counting systems do so and users might mistakenly assume that this command does the same. Including additional detail beyond that is your choice; I do not have a strong opinion on documenting that.